src/components/templates/PosterPage/PosterPage.scss

```diff
@@ -80,12 +89,11 @@ $poster-cell-optimal-height: 195px;
       background-color: $dark;
     }
   }
-  &__listeners {
-    grid-area: listeners;
 
-    .userlist-container {
-      margin-left: unset;
-      margin-right: unset;
-    }
+  div.PosterPage__listeners {
+    grid-area: listeners;
+    margin-left: unset;
+    margin-right: unset;
+    margin-bottom: $spacing--lg;
   }
```

Comment:
Looks like we still need to add a comment here explaining the 'why' of it (namely, that we are using this to increase our css specificity to override the thing). Described in more detail in the previous thread about this.

Reply:
Added comment in https://github.com/sparkletown/sparkle/pull/1299/commits/9969868e59dce1c2b5fab43539e5b4f4dbbdbfff